test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -22,52 +28,187 @@ Class. We use :proc:`~UnitTest.runTest` to pass the tests.
 :proc:`~Test.assertGreaterThan`
 :proc:`~Test.assertLessThan`
 
+Examples
+=========
+
+Basic Usage
+-------------
+
+Here is a minimal example demonstrating how to use the UnitTest module:
+
+<celsius2fahrenheit example>
```

Comment:
Sorry, I didn't mean the literal `<example>` text, but rather that as a placeholder for where you should put the example (which is below).

I'd just remove these lines.